Importer/Interpreter/Exception/FieldValueException.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Netdudes\ImporterBundle\Importer\Interpreter\Exception;
+
+class FieldValueException extends InterpreterException
```

Comment:
For me the current name is better and more specific since the logic of the interpreter, is to handle each field of each row, and build a message per field when possible (Boolean and DateTime fields)